frontend/src/components/Stores.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="container">
+    <div style="min-width:1115px" class="card mt-5">
+      <h2 class="card-header text-center">Store Information</h2>
+      <div style="text-align: center" class="m-3" v-cloak>
```

Comment:
is there a reason to have these styles inline?

Reply:
There's a bad reason only :|. Not very used to frontend development. Moved them to CSS on the component style section. Also learned about Vue scoped styles to prevent unwanted/uncontrolled cascading too. Will do better next time!

Reply:
in this case you could have used the bootstrap css utils for aligning the text for example